### PR TITLE
chore: listen to cd.yml's events

### PR DESCRIPTION
**Pull request by IvanJobs**
listen to CD's workflow events.
When new cli versions released from dev/main branch, send notifications to TeamsFx-CICD-Test.
Currently, workflows triggered in TeamsFx-CICD-Test do nothing (Will be enabled later on).

### CODE REVIEW

File: .github/workflows/cicd_test_main.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  TriggerCICDTestFromDev:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Trigger TeamsFx-CICD-Test
+        env:
+            TEAMSFX_CICD_TOKEN: ${{ secrets.TEAMSFX_CICD_TOKEN }}
+        run: |
+            curl \
+                -u IvanJobs:${TEAMSFX_CICD_TOKEN} \
```
Comment (LongOddCode) [inline .github/workflows/cicd_test_main.yml:19]: is user necessary?

File: .github/workflows/cicd_test_dev.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  TriggerCICDTestFromDev:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Trigger TeamsFx-CICD-Test
+        env:
+            TEAMSFX_CICD_TOKEN: ${{ secrets.TEAMSFX_CICD_TOKEN }}
+        run: |
+            curl \
+                -u IvanJobs:${TEAMSFX_CICD_TOKEN} \
```
Comment (LongOddCode) [inline .github/workflows/cicd_test_dev.yml:19]: -u :${{ secrets.GITHUB_TOKEN }}
this token is managed by Wei's team centrally.

**Review by LongOddCode — DISMISSED** [review]

File: .github/workflows/cicd_test_main.yml (hunk 1 above)
Comment (IvanJobs) [reply .github/workflows/cicd_test_main.yml]: Updated.

File: .github/workflows/cicd_test_dev.yml (hunk 2 above)
Comment (IvanJobs) [reply .github/workflows/cicd_test_dev.yml]: Thanks.

**Review by wenytang-ms — APPROVED** [review]

**Review by qinezh — APPROVED** [review]

**Pull request merged**